src/ingestion/kafka-queue.ts

```diff
@@ -167,21 +167,21 @@ export class KafkaQueue implements Queue {
         return await startPromise
     }
 
-    async pause(): Promise<void> {
+    pause(): void {
```

Comment:
Why not keep this `async`, just not `await`ing in practice?

Reply:
sure, reverted pausing to being async